examples/conv-opt/comparison/gen_conv_models.py

```diff
@@ -0,0 +1,88 @@
+import onnx
```

Comment:
Add an "onnx" prefix and use the hyphen "-" in the file name for consistency.

Reply:
Added onnx prefix for helper and make_tensor_value_info. Unfortunately, onnx doesn't export the numpy_helper attribute.